novelai/novelai.py

```diff
@@ -140,28 +152,76 @@ async def novelai(self,
                       sampler_version: Optional[str],
                       noise_schedule: Optional[str],
                       decrisper: Optional[bool],
+                      model: Optional[ImageModel],
+                      reference_image1: Optional[discord.Attachment],
+                      reference_image_strength1: Optional[app_commands.Range[float, 0.0, 1.0]],
+                      reference_image_info_extracted1: Optional[app_commands.Range[float, 0.0, 1.0]],
+                      reference_image2: Optional[discord.Attachment],
+                      reference_image_strength2: Optional[app_commands.Range[float, 0.0, 1.0]],
+                      reference_image_info_extracted2: Optional[app_commands.Range[float, 0.0, 1.0]],
+                      reference_image3: Optional[discord.Attachment],
```

Comment:
Is 3 vibe images necessary? That's 9 new parameters, which makes me a bit uncomfortable. If possible I would keep it to 2, but I'll trust your judgement.
Same for img2img.